src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  FC,
+  memo,
+  useEffect,
+  useState,
+} from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage:FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [isTableLoading, setIsTableLoading] = useState(false);
+  const loadPeopleFromServer = async () => {
+    try {
+      setIsTableLoading(true);
+
+      const peopleFromServer = await getPeople();
+      const preparedPeople = peopleFromServer.map(person => {
+        const mother = peopleFromServer.find(p => person.motherName === p.name);
```

Comment:
It would be better not to use a one-letter name for a variable, but it's up to you